src/main/java/net/datafaker/service/RandomService.java

```diff
@@ -69,6 +67,36 @@ public double nextDouble(double min, double max) {
         return min + (nextDouble() * (max - min));
     }
 
+    /**
+     * Gaussian or normal distribution.
+     * @see <a href="https://en.wikipedia.org/wiki/Normal_distribution">Normal distribution</a>
+     * ~ 68 % of generated values will belong to the segment [mean - deviation, mean + deviation]
+     * ~ 95 % of generated values will belong to the segment [mean - 2 * deviation, mean + 2 * deviation]
+     * ~ 99 % of generated values will belong to the segment [mean - 3 * deviation, mean + 3 * deviation]
+     *
+     * @return Returns the next pseudorandom, Gaussian ("normally") distributed double value with mean 0.0
+     * and standard deviation 1.0 from this random number generator's sequence.
+     */
```

Comment:
We do not master this... and we do not track if anything is changing between jdk's versions
That's why it seems it's better to refer to jdk's javadoc